schema/index.js

```diff
@@ -98,7 +98,7 @@ const Viewer = {
 
 const schema = new GraphQLSchema({
   mutation: new GraphQLObjectType({
-    name: "RootMutationType",
+    name: "Mutation",
```

Comment:
This one is not actually hardcoded, as far as I could tell, but to keep it consistent I renamed this one too to the conventional name.